index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=
```

Comment:
You have mistakingly modified this meta tag making the tag not close up properly. This makes the website not render correctly. Could you have it corrected?